tests/testthat/test-add_co2-summarize_co2_range.R

```diff
@@ -0,0 +1,117 @@
+test_that("different benchmarks output different number of rows", {
+  x <- tidyr::expand_grid(
+    benchmark = c("all", "unit", "tilt_sector", "unit_tilt_sector"),
+    emission_profile = c("low", "medium", "high"),
+    unit = c("m2", "kg"),
+    tilt_sector = c("sector1", "sector2"),
+    tilt_subsector = c("subsector1", "subsector2"),
+  )
+  y <- tibble(
+    emission_profile = c("low", "medium", "high"),
+    isic_4digit = "'1234'",
+    co2_footprint = 1:3,
+  )
+  data <- left_join(x, y, by = col_risk_category_emissions_profile(), relationship = "many-to-many")
+
+  benchmark <- "all"
+  expected <- 3
+  # 3 = 3 emission_profile
+  out <- summarize_co2_range(data)
+  expect_equal(nrow(filter(out, benchmark == .env$benchmark)), expected)
+
+  benchmark <- "unit"
+  expected <- 6
+  # 6 = 3 emission_profile * 2 unit
+  out <- summarize_co2_range(data)
+  expect_equal(nrow(filter(out, benchmark == .env$benchmark)), expected)
```

Comment:
@maurolepore I am a bit confused here. Why you used `.env$benchmark` here and not just `benchmark`?

Reply:
Thanks for the info! 👍